docs/operation/volume_workflow/storageclass_creation.rst

```diff
@@ -0,0 +1,49 @@
+StorageClass Creation
+=====================
+**MetalK8s** makes use of StorageClasses in other to provision and use
```

Comment:
`**MetalK8s** uses **StorageClass** to configure how **Volume** are formatted and mounted`